client/src/tpu_client.rs

```diff
@@ -13,21 +13,13 @@ use {
         transport::Result as TransportResult,
     },
     solana_tpu_client::tpu_client::{Result, TpuClient as BackendTpuClient},
-    solana_udp_client::{UdpConfig, UdpConnectionManager, UdpPool},
     std::sync::Arc,
 };
 pub use {
     crate::nonblocking::tpu_client::TpuSenderError,
     solana_tpu_client::tpu_client::{TpuClientConfig, DEFAULT_FANOUT_SLOTS, MAX_FANOUT_SLOTS},
 };
 
-pub type QuicTpuClient = TpuClient<QuicPool, QuicConnectionManager, QuicConfig>;
-
-pub enum TpuClientWrapper {
-    Quic(TpuClient<QuicPool, QuicConnectionManager, QuicConfig>),
-    Udp(TpuClient<UdpPool, UdpConnectionManager, UdpConfig>),
-}
```

Comment:
Just confirming: both these pub definitions were added recently, only in master/v2, correct?

Reply:
you are correct. these were NOT backported to 1.18